cypress/cypress-smoketests/support/email-receipt.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  DeleteObjectsCommand,
+  GetObjectCommand,
+  ListObjectsV2Command,
+  S3Client,
+} from '@aws-sdk/client-s3';
+
+const s3 = new S3Client({
+  region: 'us-east-1',
+});
+
+export const deleteAllItemsInEmailBucketWithRetries = async ({
+  bucketName,
+  retries,
+}: {
+  bucketName: string;
+  retries?: number;
+}) => {
+  if (retries) {
+    try {
+      return await deleteAllItemsInEmailBucket(bucketName);
+    } catch (error) {
+      console.error(
+        'Error with deleting items in email bucket. Retrying. Error: ',
+        error,
+      );
+
+      setTimeout(() => {}, 2000);
```

Comment:
do we need to make sure it waits before retrying? Should we wrap the timeout in a promise to make sure it waits?

Reply:
Just saw this, agreed!